### PR TITLE
Add tableContains helper

### PR DESCRIPTION
**Pull request by thec0keman**

### CODE REVIEW

File: addon-test-support/helpers/table-contains-helper.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      if (isBlank(rowRule)) {
+        return;
+      }
+
+      rowRule.forEach((cellRule, column) => {
+        if (isBlank(cellRule)) {
+          return;
+        }
+
+        const cellSeletor = `tr:eq(${ row })>td:eq(${ column })`;
```
Comment (supremebeing7) [inline addon-test-support/helpers/table-contains-helper.js]: typo - `cellSelector`

File: tests/unit/helpers/assert-helpers-test.js [hunk 2]
```diff
@@ -120,3 +120,105 @@ test('assert.classNotIncludes / assert.classNotContains', function(assert) {
...
+          <td>4</td>
+        </tr>
+      </tbody>
+    </table>
+  `);
+
+  assert.tableContains(this.$('table:eq(0)'), {
+    body: [
+      [1,2],
+      [null, 4]
```
Comment (supremebeing7) [inline tests/unit/helpers/assert-helpers-test.js]: Does this pass? I must be misunderstanding... on line 161, there is 3 in that cell.

Comment (supremebeing7) [inline tests/unit/helpers/assert-helpers-test.js]: Oh I think I understand. So, if you only want to check certain values are present, you can just put null for the others?

Comment (thec0keman) [inline tests/unit/helpers/assert-helpers-test.js]: yeah.  I thought about adding a '*' character or something, but I also liked the simplicity of  'cells I don't want to test, I don't send a value'.   What do you think?

Comment (supremebeing7) [inline tests/unit/helpers/assert-helpers-test.js]: yeah that makes sense.

**Comment by supremebeing7** [comment]
Suggest updating the README as well.

File: addon-test-support/helpers/table-contains-helper.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { isBlank } from 'ember-utils';
+
+function buildError(message, actual, row, column) {
+  return `Attempted to ${ message } at table row ${ row }, column ${ column }, but ${ actual} was found`;
```
Comment (ekluff) [inline addon-test-support/helpers/table-contains-helper.js]: Insert space after 'actual' for consistent formatting

**Pull request merged**